AAUnlimited/MemMods/AAPlay/Events/Loads.cpp

```diff
@@ -106,12 +109,24 @@ DWORD __declspec(noinline) __stdcall CallOrigLoad(DWORD who, void *_this, DWORD
 		call dword ptr[who]
 		mov retv, eax
 	}
-	auto card = AAPlay::g_characters[loadCharacter->m_seat].m_cardData;
-
-	for (int idx = 0; idx < 4; idx++) {
-		if (card.GetHairs(idx).size()) {
-			for (int num = 0; num < card.GetHairs(idx).size(); num++) {
-				AAPlay::g_characters[loadCharacter->m_seat].AddShadows((DWORD*)AAPlay::g_characters[loadCharacter->m_seat].m_hairs[idx][num].second);
+	
+	if (General::IsAAPlay) {
+		auto card = AAPlay::g_characters[loadCharacter->m_seat].m_cardData;
+		for (int idx = 0; idx < 4; idx++) {
+			if (card.GetHairs(idx).size()) {
+				for (int num = 0; num < card.GetHairs(idx).size(); num++) {
+					AAPlay::g_characters[loadCharacter->m_seat].AddShadows((DWORD*)AAPlay::g_characters[loadCharacter->m_seat].m_hairs[idx][num].second);
+				}
+			}
+		}
+	}
+	if (General::IsAAEdit) {
+		AAEdit::g_currChar.m_char = loadCharacter;
+		for (int idx = 0; idx < 4; idx++) {
+			if (AAEdit::g_currChar.m_cardData.GetHairs(idx).size()) {
+				for (int num = 0; num < AAEdit::g_currChar.m_cardData.GetHairs(idx).size(); num++) {
+					AAEdit::g_currChar.AddShadows((DWORD*)AAEdit::g_currChar.m_hairs[idx][num].second);
+				}
```

Comment:
these two can be simplified into one. Just pick the character based on IsAAPlay or IsAAEdit